src/main/resources/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorBuildStep/help-advancedProperties.html

```diff
@@ -0,0 +1,22 @@
+<!--
+
+    Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+
+    This program is licensed to you under the Apache License Version 2.0,
+    and you may not use this file except in compliance with the Apache License Version 2.0.
+    You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the Apache License Version 2.0 is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+
+-->
+<div>
+  Provide advanced properties to the IQ Server in &lt;key&gt;=&lt;value&gt; pairs, with
+  each additional property appearing on a new line. eg: <br/>
+  <code>
+    dirIncludes=**/.*<br/>
+    fileExcludes=target/my.jar
```

Comment:
Unless you explicitly hear otherwise from Jamie or Justin, please remove these specific example properties.

The help in the both the old Hudson plugin and the Bamboo plugin makes it clear that those props are "to be set on a case-by-case basis as advised by Sonatype Support" (https://github.com/sonatype/insight-ci/blob/master/nexus-iq-jenkins-plugin/src/main/webapp/help/advancedOptions.html) and I'd like to keep that notion. The two properties you mention here invite customers to experiment with them given their enticing names and set them up for failure due to misunderstanding.

If we want to give an example of the line-based format, we should rather use generic properties like "key1=value1" and "key2=value2".

Reply:
Yes, it also is implied support for these additional properties which AFAIK are unadvertised.